lib/resources/security_policy.rb

```diff
@@ -149,7 +149,7 @@ def extract_value(val)
       if val =~ /^\d+$/
         val.to_i
       # special handling for SID array
-      elsif val =~ /[,]{0,1}\*\S/
+      elsif val =~ /[,]{0,1}\*\S|\,\D\S*$/
```

Comment:
Same. The original and your change are both wrong for the given need. I don't understand the use of `\D` and `\S*` can match nothing. It also seems that the `$` isn't necessary. 

This change does not have a test to show how it is used, so I can't really comment further.

--

Mismo. El original y su cambio son incorrectos para la necesidad dada. No entiendo que el uso de `\D` y `\S *` no puede coincidir con nada. También parece que el `$` no es necesario.

Este cambio no tiene una prueba para mostrar cómo se usa, por lo que realmente no puedo comentar más.